tests/data_context/store/test_store_backends.py

```diff
@@ -1204,6 +1204,49 @@ def test_TupleGCSStoreBackend():  # noqa: PLR0915
     )
 
 
+@pytest.mark.big
```

Comment:
Non blocking - but does this test need to be marked big and not unit? It looks like relevant items are mocked.

Reply:
Great catch. Re-marking before merging :)